source/guide_gollum.rst

```diff
@@ -0,0 +1,150 @@
+
+Now that CMake is installed, we need to make sure, that Gollum will use the new CMake version for its own install process. Therefore we need to add the new CMake binary to the PATH variable temporarily.
+
+::
+
+  [isabell@stardust ~]$ export PATH=$PATH:/home/isabell/cmake-3.17.3-Linux-x86_64/bin/)
+  [isabell@stardust ~]$
+
+Now make sure that your new CMake version works correctly by checking the version number using this command:
+
+::
+
+  [isabell@stardust ~]$ cmake --version
+  cmake version 3.17.3
+  [isabell@stardust ~]$
+
+
+All good! Now we're ready to install Gollum.
+
+Create directory for Gollum content
+-----------------------------------
+
+All the data within Gollum is kept in one directory, which is also where the underlying Git repository is located. You should create a new, empty directory in your home folder if you want to start a new wiki (or clone an existing Git repository instead).
+
+::
+
+  [isabell@stardust ~]$ mkdir wiki
+  [isabell@stardust ~]$
+
+
+Install Gollum
+=============
+
+Gollum can now be installed via the ruby package manager by running:
+
+::
+
+  [isabell@stardust ~]$ gem install gollum
+  [isabell@stardust ~]$
+
+You can now remove CMake (both the installer and the folder) again, if you want to. You should also edit your ``.bash_profile`` to remove the CMake folder from the ``PATH`` variable.
+
+::
+
+  [isabell@stardust ~]$ rm -rf cmake-3.17.3-Linux-x86_64.sh cmake-3.17.3-Linux-x86_64
+  [isabell@stardust ~]$
+
+
+Now you should be able to start Gollum via:
+
+::
+
+  [isabell@stardust ~]$ gollum /home/isabell/wiki
+  [isabell@stardust ~]$
+
+However Gollum is not yet reachable from the internet. So, close the Gollum application again (by pressing ``CTRL + C``).
+
+Setting up a web backend
+------------------------
+
+To be able to reach the Gollum appliaction from the internet, we need to wire up the proper ports via :manual:`Uberspace web backends <web-backends>`. By default Gollum runs on port 4567, so we need to create a web backend for this port. If you want to reach Gollum using your regular Uberspace base URL (e.g. ``https://isabell.uber.space/``), just run:
+
+::
+
+  [isabell@stardust ~]$ uberspace web backend set / --http --port 4567
+  Set backend for / to port 4567; please make sure something is listening!
+  You can always check the status of your backend using "uberspace web backend list".
+  [isabell@stardust ~]$
+
+.. note:: If you want to run Gollum under a different URL, just adjust the "/" parameter in the command above to match your preferred path.
+
+
+Now you can start Gollum again and you should be able to open it in your browser using your Uberspace URL.
```

Comment:
To make the guides more maintainable there are prepared snippets.
```suggestion
.. note:: By default Gollum runs on port 4567, so we need to create a web backend for this port. 
.. include:: includes/web-backend.rst

.. note:: If you want to run Gollum under a different URL, just adjust the "/" parameter in the command above to match your preferred path.

Now you can start Gollum again and you should be able to open it in your browser using your Uberspace URL (e.g. ``https://isabell.uber.space/``).
```

the snippet for `.. include:: includes/web-backend.rst` currently is:
```
To make the application accessible from the outside, configure a `web backend <webbackend_>`_:

::

  [isabell@stardust ~]$ uberspace web backend set / --http --port <port>
  Set backend for / to port <port>; please make sure something is listening!
  You can always check the status of your backend using "uberspace web backend list".
  [isabell@stardust ~]$

.. _webbackend: https://manual.uberspace.de/web-backends.html

```